base/docs/helpdb/Base.jl

```diff
@@ -2438,19 +2438,11 @@ the process.
 triu!(M, k)
 
 """
-    readall(stream::IO)
+    readstring(stream or filename)
 
-Read the entire contents of an I/O stream as a string.
+Read the entire contents of an I/O stream or a file as a string.
```

Comment:
No.
A filename has no concept of open or close.
The doc says that it reads the entire contents of a file. That is what it does, open/close is hidden implementation detail. A stream may have a concept of open/close but if it does that is beyond the scope of this function, all this function does is read as a string.